lib/addons/karpenter/index.ts

```diff
@@ -87,6 +87,17 @@ interface KarpenterAddOnProps extends HelmAddOnUserProps {
      * Only applicable for v0.19.0 and later
      */
     interruptionHandling?: boolean,
+
+    /**
+     * Limits define a set of bounds for provisioning capacity.
+     */
+    limits?: {
+        resources?: {
+          cpu?: number;
+          memory?: string;
+          [k: string]: unknown;
```

Comment:
We may also document this part - referring to the specific instance type (i.e. GPU)